### ISSUE TITLE
Uncaught ReferenceError: fs is not defined

### ISSUE THREAD

**Issue by lmao-emerald** [h3imdall/ftp-remote-edit#122]
[Enter steps to reproduce:]

Trying to open the plugin.

**Atom**: 1.21.1 x64
**Electron**: 1.6.15
**OS**: Microsoft Windows 7 Home Premium 
**Thrown From**: [ftp-remote-edit](https://github.com/h3imdall/ftp-remote-edit) package 0.12.4

### Stack Trace

Uncaught ReferenceError: fs is not defined

```
At C:\Users\jaizu\.atom\packages\ftp-remote-edit\lib\helper\helper.js:44

ReferenceError: fs is not defined
    at getConfig (/packages/ftp-remote-edit/lib/helper/helper.js:44:3)
    at checkPasswordExists (/packages/ftp-remote-edit/lib/helper/secure.js:31:22)
    at FtpRemoteEdit.toggle (/packages/ftp-remote-edit/lib/ftp-remote-edit.js:143:12)
    at HTMLElement.ftpRemoteEditToggle (/packages/ftp-remote-edit/lib/ftp-remote-edit.js:41:44)
    at CommandRegistry.handleCommandEvent (~/AppData/Local/atom/app-1.21.1/resources/app/src/command-registry.js:381:36)
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (~/AppData/Local/atom/app-1.21.1/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:621:22)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (~/AppData/Local/atom/app-1.21.1/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:412:28)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (~/AppData/Local/atom/app-1.21.1/resources/app/src/window-event-handler.js:100:42)
    at HTMLDocument.<anonymous> (~/AppData/Local/atom/app-1.21.1/resources/app/src/window-event-handler.js:3:65)
```

### Commands

```
     -1:12.3.0 ftp-remote-edit:toggle (atom-pane.pane)
     -0:58.5.0 ftp-remote-edit:edit-servers (atom-workspace.workspace.scrollbars-visible-always.theme-one-light-syntax.theme-one-light-ui)
  2x -0:55.6.0 core:confirm (input.hidden-input)
     -0:52.5.0 core:cancel (input.hidden-input)
     -0:10.9.0 fuzzy-finder:toggle-file-finder (div.package-detail.panels-item)
     -0:09.9.0 editor:consolidate-selections (input.hidden-input)
     -0:09.9.0 core:cancel (input.hidden-input)
     -0:09.1.0 application:new-file (div.package-detail.panels-item)
     -0:08.4.0 ftp-remote-edit:toggle (input.hidden-input)
```

### Non-Core Packages

```
ftp-remote-edit 0.12.4 
```

**Comment by kweensey** [comment]
Same here.

**Comment by NoneTheLess05** [comment]
Me too

**Comment by fufales** [comment]
Yeah :)

**Comment by tarutin** [comment]
Some problem.

**Comment by tristanfazio** [comment]
roll back to the previous version with apm install ftp-remote-edit@0.12.3 for now guys!

**Comment by dlhines** [comment]
Greatful...

**Comment by cschindl** [comment]
Error was fixed fb885ea727d0adac516a010466e4a24951e28917.
